src/components/DropArea/SbDropArea.vue

```diff
@@ -0,0 +1,221 @@
+<template>
+  <div
+    class="sb-drop-area"
+    :class="{ 'sb-drop-area__over': isOver }"
+    @dragover="dragOver"
```

Comment:
You could rename the listeners with `handle` prefix. For example: `dragOver` -> `handleDragOver`